### PR TITLE
DISCOVERY-222 - Update ansible version

### PR DESCRIPTION
**Pull request by bruno-fs**
Throw away code that wasn't preventing this migration.

Besides getting up to date with ansible, this also fixes DISCOVERY-183 and DISCOVERY-177.

### CODE REVIEW

**Comment by bruno-fs** [comment]
Huh, I might need to take a step back and reconsider removing `exclude hosts` option. That's an actual option in use on the CLI

```% qpc source add --help                                                                                                                            ✹
usage: qpc source add [-h] --name NAME --type TYPE --hosts HOSTS [HOSTS ...] [--exclude-hosts EXCLUDE_HOSTS [EXCLUDE_HOSTS ...]] --cred CRED
                      [CRED ...] [--port PORT] [--ssl-cert-verify {True,False,true,false}] [--ssl-protocol {SSLv23,TLSv1,TLSv1_1,TLSv1_2}]
                      [--disable-ssl {True,False,true,false}] [--use-paramiko {True,False,true,false}]

optional arguments:
  -h, --help            show this help message and exit
  --name NAME           Source name.
  --type TYPE           Type of source. Valid values: vcenter, network, satellite.
  --hosts HOSTS [HOSTS ...]
                        IP ranges to scan. Run the "man qpc" command for more information about supported formats.
  --exclude-hosts EXCLUDE_HOSTS [EXCLUDE_HOSTS ...]
                        IP ranges to exclude from scan. Only supported for network sources. Run the "man qpc" command for more information about
                        supported formats.
  --cred CRED [CRED ...]
                        Credentials to associate with a source.
  --port PORT           Port to use for connection for the scan; network default is 22, vcenter default is 443.
  --ssl-cert-verify {True,False,true,false}
                        If true, the SSL certificate will be verified when making requests to the source, otherwise no verification will occur. Not
                        valid for network sources.
  --ssl-protocol {SSLv23,TLSv1,TLSv1_1,TLSv1_2}
                        The SSL protocol to be used during a secure connection. Not valid for network sources.
  --disable-ssl {True,False,true,false}
                        Disable SSL usage during a connection. Not valid for network sources.
  --use-paramiko {True,False,true,false}
                        Set Ansible connection method to paramiko.default connection method is ssh.
                        ````

**Comment by bruno-fs** [comment]
Removing review request. this still needs some work

**Comment by bruno-fs** [comment]
`exclude_hosts` option restored. ready to review

File: Dockerfile [hunk 1]
```diff
@@ -7,8 +7,8 @@ ENV DJANGO_LOG_FORMATTER=verbose
 ENV DJANGO_LOG_HANDLERS=console,file
 ENV DJANGO_LOG_LEVEL=INFO
 ENV DJANGO_SECRET_PATH=/var/data/secret.txt
-ENV LANG=C
-ENV LC_ALL=C
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
```
Comment (bruno-fs) [inline Dockerfile:11]: I'll drop this commit after review. We don't need this change.

**Review by gtanzillo — APPROVED** [review]
LGTM 👍 Nice cleanup with this change!

**Review by nicolearagao — APPROVED** [review]
nice 👍🏼

**Pull request merged**